sotodlib/site_pipeline/calibration/wiregrid.py

```diff
@@ -418,18 +418,21 @@ def get_cal_gamma(tod, wrap_aman=False):
         _det_angle_err.append(np.sqrt(_cal_data.Uerr.T[:,_i]**2 + _cal_data.Qerr.T[:,_i]**2))
     _det_angle = np.array(_det_angle).T
     _det_angle_err = np.array(_det_angle_err).T
+    _cal_amp = _cfit_result.cr
     _bg_theta = 0.5*np.arctan2(_cfit_result.cy0, _cfit_result.cx0)%np.pi - np.nanmean(_det_angle%np.pi, axis=1)
     _bg_amp = np.sqrt(_cfit_result.cx0**2 + _cfit_result.cy0**2)
+    if remove_cal_data: tod.move('wg', None)
     if wrap_aman:
         if 'gamma_cal' in dir(tod): tod.move('gamma_cal', None)
         _gamma_ax = core.AxisManager(tod.dets)
         _gamma_ax.wrap('gamma_raw', _det_angle, [(0, 'dets')])
```

Comment:
What is the difference between `gamma_raw` and `gamma` that is not described anywhere.

Reply:
Yeah, the raw output of gamma is a bit technical. This includes the dependencies on the steps of the wire grid. I didn't assume people use them cause it's a bit confusing for the people who want to know just the calibrated values. But I'll add descriptions about that for people who are interested in the wire grid analysis.